### ISSUE TITLE
alternative project

### ISSUE THREAD

**Issue by artmg** [ha-warmup/warmup#21]
Hi,

you may be interested to see an alternative project that has been developed. Although it does not yet have all the features of this one, it does already have a [pypi release](https://pypi.org/project/pywarmup) and a [prototype branch for the HA integration](https://github.com/lkollar/home-assistant/tree/warmup).

I was wondering if it might make sense to capitalise on the progress on that project, and see if we can get the extra features we already have integrated into it? 

I am actively using override so I would be keen to make sure that feature was implemented. 

What do you think @foxy82 @rct ?

**Comment by foxy82** [comment]
I'm easy. In some ways it depends how willing the other project is for contribution. I don't think it would take much work our end to push a pypi package but the more the merrier if they want our contribution I'm happy.

**Comment by rct** [comment]
@artmg - If there is any way to merge efforts with @lkollar and anyone else who is interested in working on this, that sounds like a great idea to me.  I can't see any benefit to having multiple projects for one thermostat.

Though, I think I'd try to base the pypi release off *this* repo, since the work has already been done to decouple the project from a single developer who might not be available all of the time. 

On a side note, the current state of this project (need to check when I last pulled it down) is working well for me to keep track of my 4 thermostats so I've been pretty happy.

**Comment by rct** [comment]
On a related note, does anyone have a contact with @alex-0103, so that maybe the README on his repo could get updated to point to this repo, or which ever one is most current?

**Comment by artmg** [comment]
good idea @rct - I have raised [PR9 on his repo](https://github.com/alex-0103/warmup4IE/pull/9) which should allow him to include a link to this project with minimal effort

**Comment by artmg** [comment]
I have not found one yet that is more recent, I'm afraid

**Comment by foxy82** [comment]
Hi. I still use this but I use it to display and log current settings I've not really used it for control.
